src/Ninja/G1CP/Content/Testsuite/getter.d

```diff
@@ -0,0 +1,120 @@
+/*
+ * Check if item exists and return its symbol index
+ */
+func int G1CP_Testsuite_GetItemId(var string name) {
+    // Check if item instance exists
+    if (!G1CP_IsItemInst(name)) {
+        var string msg; msg = ConcatStrings("Item '", name);
+        msg = ConcatStrings(msg, "' not found");
+        G1CP_TestsuiteErrorDetail(msg);
+        G1CP_TestsuiteStatusPassed = FALSE;
+    };
+
+    // Return the item's symbol index
+    return MEM_GetSymbolIndex(name);
+};
+
+/*
+ * Check if integer variable exists and return its symbol pointer
+ */
+func int G1CP_Testsuite_GetIntVarPtr(var string name, var int arrIdx) {
```

Comment:
I think we don't need that function. `G1CP_Testsuite_CheckIntVar` will probably suffice. The pointer is not required, because we can read/write the variable using `G1CP_GetVar` and `G1CP_SetVar`.